test/test_va_api_query_vendor.cpp

```diff
@@ -26,28 +26,25 @@
 
 namespace VAAPI {
 
-// no addtl. fixture functionality needed... just a unique test name
+/*
+ Inheriting from VAAPIFixture is not necessory as this test is not using any
+ variables or functions defined in fixture. A unique test name is sufficient.
+ */
```

Comment:
Please use `//` comment style here.  Misspelled `necessary` .

Also, 

> ...this test is not using any variables or functions defined in fixture...

This is not a true statement, the test uses `getDisplay` from the fixture.  Nonetheless, even if this statement was true, then that still is not the reason for using typedef instead of inheritance.  The fact is that this test does not need to override or extend the fixture, therefore a typedef is sufficient to create a unique test name/scope for gtest.

Reply:
Got it, updated changed patch.